src/Mono.Android/Android.Runtime/JNINativeWrapper.cs

```diff
@@ -45,6 +36,15 @@ public static Delegate CreateDelegate (Delegate dlg)
 
 			get_runtime_types ();
 
+			var delegateType = dlg.GetType ();
+			var result = CreateBuiltInDelegate (dlg, delegateType);
+			if (result != null)
+				return result;
+
+			if (JNIEnv.LogAssemblyCategory) {
+				JNIEnv.monodroid_log (LogLevel.Warn, LogCategories.Default, $"Falling back to System.Reflection.Emit for delegate type '{delegateType}': {dlg.Method}");
```

Comment:
Shouldn't this be `LogLevel.Debug` or `.Info`, not `.Warn`?

Reply:
Should this really be using `LogCategories.Default`, and not `.Assembly`?

Reply:
Yeah, should probably be:
```suggestion
				JNIEnv.monodroid_log (LogLevel.Debug, LogCategories.Assembly, $"Falling back to System.Reflection.Emit for delegate type '{delegateType}': {dlg.Method}");
```